.github/workflows/cloud-deployment-test.yml

```diff
@@ -47,6 +47,7 @@ jobs:
       - uses: actions/checkout@v4
         with:
           repository: 'CorfuDB/corfudb-cloud'
+          ref: master
```

Comment:
just wondering isn't this the default?

Reply:
Yes, It is the default, but it helps to have this field specifically to avoid any ambiguity for branch-specific forks. for example, `0.4.2.0` release branch should have its branch name here in place of `master`.